src/LightningExtension/Context/UtilityContext.behat.inc

```diff
@@ -27,33 +29,6 @@ class UtilityContext extends DrupalSubContextBase {
   }
 
   /**
-   * Asserts that a certain number of elements match a CSS selector.
-   *
-   * @param string $selector
-   *   The selector.
-   * @param int $count
-   *   The number of elements expected to match the selector.
-   *
-   * @throws ExpectationException
-   *   If the number of elements that match the selector is not the expected
-   *   number.
-   *
-   * @Then :count element(s) should match :selector
-   */
-  public function assertSelectorMatch($selector, $count) {
```

Comment:
I removed this because it turns out that the Mink extension already has a step definition that does the same thing. So this counts as gardening.